spec/helpers/search_helper_spec.rb

```diff
@@ -257,16 +257,20 @@
         expect(make_summary_p(search)).to eq('<p>1 resultado</p>')
       end
 
-      xit "should return 'Página %{page} de %{total} resultados' when total is 2..99 and page > 1" do
+      it "should return 'Página %{page} de %{total} resultados' when total is 2..99 and page > 1" do
         search = double(Search, :total => 80, :page => 5, :first_page? => false)
         expect(make_summary_p(search)).to eq('<p>Página 5 de 80 resultados</p>')
       end
 
-      xit "should return 'Página %{page} de aproximadamente %{total} resultados' when total >= 100 and page > 1" do
+      it "should return 'Página %{page} de aproximadamente %{total} resultados' when total >= 100 and page > 1" do
         search = double(Search, :total => 2000, :page => 5, :first_page? => false)
         expect(make_summary_p(search)).to eq('<p>Página 5 de aproximadamente 2,000 resultados</p>')
       end
 
+      it "should pluralize spanish words for spanish language" do
```

Comment:
@peggles2 , because the `pluralize` method could be called potentially anywhere in our code, this spec should be moved somewhere higher level. It's not specific to one specific SearchHelper method.  I'm thinking something like `spec/internationalization_spec.rb` (which would be new). Also, you've added 6 different inflection rules, but one spec; it would be better to test each rule.

Reply:
Ok added a tests for the added Spanish inflections that should cover this as well.